lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -96,7 +96,13 @@ protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
      */
     protected function doContains($id)
     {
-        return $this->redis->exists($id);
+        $exists = $this->redis->exists($id);
+
+        if (\is_bool($exists)) {
```

Comment:
This needs to be `use function is_bool;` at the top of the file. You can automatically handle this by running the following command:

```
./vendor/bin/phpcbf
```

Reply:
Wouldn't use phpcbf due #265. :/

Reply:
@Majkl578 So I just have to wait #265 to be merged ?

Reply:
No, use `use function is_bool` and it's good to go.

Reply:
Now that the use statement is there, you can remove the `\` before the ``is_bool`` call.